File: Vivu/php/hoi - app/check.php
<?php
// IT-Team:vivu
//$id=$_REQUEST['id'];
$db=mysqli_connect("localhost","root","","DB_thaoluan");
/*$view=mysqli_query($db,"slect view_q from id where id=$id and ok like 'n'");
if ($view){
    while ($row=mysqli_fetch_array($view)){
        $datav=$row['view_q'];
    }
    echo 'y '+$datav;
}else{
    
    
}
*/
$query="SELECT stt FROM awso";
$con=mysqli_query($db,$query);
$data=array();
if(mysqli_num_rows($con)){
    while ($row=mysqli_fetch_array($con,MYSQLI_NUM)){
          $data[]=$row;
    }
    echo json_encode($data);
}else{
    echo "n";
}
mysqli_free_result($con);
mysqli_close($db);
?>

File: Vivu/Nodejs/chat/old/check.py
<?php

$host = 'localhost';
$user = 'root';
$pass = '';
$db = 'web_bt';
$con=mysqli_connect($host, $user, $pass,$db)or die ('Không the ket noi toi database');


$result=mysqli_query($con,"SELECT * FROM cauhoi");

$row=mysqli_fetch_array($result);
echo $row;

?>